app/views/incomeSources/manage/ManageSelfEmployment.scala.html

```diff
@@ -0,0 +1,176 @@
+        section = Some(getMessage("sole-trader-section")),
+        captionClasses = Some("govuk-caption-l")
+    ))
+
+    <dl class="govuk-summary-list" id="refund-to-taxpayer-table">
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">@getMessage("business-name")</dt>
+            <dd class="govuk-summary-list__value" id="business-name">@{
+                extractOption(viewModel.tradingName)
+            }</dd>
+        </div>
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">@getMessage("business-address")</dt>
+            <dd class="govuk-summary-list__value" id="business-address">@{
+                viewModel.address match {
+                    case Some(x) => HtmlFormat.raw(x)
+                    case None => messages("incomeSources.generic.unknown")
+                }
+            }</dd>
+        </div>
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">@getMessage("date-started")</dt>
+            <dd class="govuk-summary-list__value" id="business-date-started">@{
+                viewModel.tradingStartDate match {
+                    case Some(x) => x.toLongDate
+                    case None => messages("incomeSources.generic.unknown")
+                }
+            }</dd>
+        </div>
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">@getMessage("accounting-method")</dt>
+            <dd class="govuk-summary-list__value" id="business-accounting-method">@{
+                viewModel.businessAccountingMethod match {
+                    case Some(x) => x match {
+                        case value if value == "cash" => getMessage("cash-accounting")
+                        case value if value == "accruals" => getMessage("traditional-accounting")
+                        case _ => throw new MissingFieldException("Unexpected value for businessAccountingMethod field")
+                    }
+                    case None => messages("incomeSources.generic.unknown")
+                }
+            }</dd>
+        </div>
+        @if(viewModel.itsaHasMandatedOrVoluntaryStatusCurrentYear.getOrElse(throw MissingFieldException("Missing itsaHasMandatedOrVoluntaryStatusCurrentYear field"))
+                && viewModel.latencyDetails.isDefined) {
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">@getMessage("reporting-method", viewModel.latencyDetails.get.taxYear1.toString, (viewModel.latencyDetails.get.taxYear2).toString)</dt>
+                <dd class="govuk-summary-list__value" id="reporting-method-1">@{
+                    viewModel.latencyDetails.get.latencyIndicator1 match {
+                        case method: String if method == "A" => getMessage("annually")
+                        case method: String if method == "Q" => getMessage("quarterly")
+                        case _ => "Field Missing"
```

Comment:
you would need read this value from the resources like: getMessage, I guess there is translation in Welsh for this

Reply:
This field should always be "Q" or "A" when coming from the api so should I just remove this line and not have a case _

Reply:
removing line now